api/types.go

```diff
@@ -336,8 +332,9 @@ func (opts *Options) FromMap(m map[string]interface{}) error {
 	}
 
 	if len(invalidOpts) > 0 {
-		return fmt.Errorf("%w: %v", ErrInvalidOpts, strings.Join(invalidOpts, ", "))
+		log.Println("warning, invalid options provided:", strings.Join(invalidOpts, ", "))
```

Comment:
This error is removed so that clients specifying the options which no longer exist continue to work, correct?  This could cause us some more pain in the future. It used to be pretty common for issues to be opened reporting that "X option does not work" due to typos in option names not being easy to notice without the error return.

Reply:
I've been thinking we should expose a little bit more "info" metadata on the response, possibly things for `/set verbose` mode (my use-case is more GPU related info), but this scenario feels like it could also be useful.  A list of "warnings" that we can expose up to the user while still proceeding.